examples/tutorials/advanced/websockets-example-MNIST/Federated learning with websockets and federated averaging.ipynb

```diff
@@ -209,63 +337,20 @@
     "print(model)"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Now we are ready to define the training loop. We will perform training over a given number of batches separately on each worker and then calculate the federated average of the resulting model."
```

Comment:
Maybe keep this information even if you remove the train function description, because it is important to understand how the training works.

Reply:
sure

Reply:
@LaRiffle done